src/tests/test_piranha_java.rs

```diff
@@ -32,14 +32,14 @@ create_rewrite_tests! {
       "treated"=>  "true",
       "treated_complement" => "false",
       "namespace" => "some_long_name"
-    };
+    }, delete_file_if_empty= false;
```

Comment:
Do we have a test case for `delete_file_if_empty = true`? Or is that the default for all other tests?

Reply:
`delete_file_if_empty = true` is default. So we don't explicitly test that.

Reply:
Do we? Because I see elsewhere (see the next comment) that we check that the count of files at `path_to_codebase` and `path_to_expected` is the same. Not sure if that check holds for _every_ test, but, if it does, then we are never actually testing that empty files are being deleted correctly...